soccer/Processor.cpp

```diff
@@ -119,7 +119,7 @@ Processor::~Processor() {
     }
 
     // DEBUG - This is unnecessary, but lets us determine which one breaks.
-    //_refereeModule.reset();
+    //    _refereeModule.reset();
```

Comment:
Can you just delete these two lines? Or uncomment them, idk what the context is.